packages/plugin-electron-net-breadcrumbs/net-breadcrumbs.js

```diff
@@ -62,3 +70,5 @@ module.exports = net => ({
     }
   }
 })
+
+const getDuration = (startTime) => startTime && new Date() - startTime
```

Comment:
I think this needs to be stricter if we expect `startTime` to be missing — right now it can return `false`, which will end up in metadata and visible in the dashboard

If `startTime` is missing, it should probably return `undefined` so that it gets omitted from the JSON payload entirely

```suggestion
const getDuration = (startTime) => startTime ? new Date() - startTime : undefined
```

Reply:
This could also use a test

Reply:
Hmm this is already returning `undefined` when `startTime` is `undefined`. The `&&` operator just returns the value of the first operand if it's falsey - it doesn't coerce it to a boolean value.

Two of the existing tests are already testing for this (albeit implicitly) because they abort the request without calling `request.end`, meaning `startTime` is `undefined`: https://github.com/bugsnag/bugsnag-js/blob/PLAT-9443/network-breadcrumb-duration/packages/plugin-electron-net-breadcrumbs/test/net-breadcrumbs.test-main.ts#L118-L162

These tests would fail if the breadcrumb's `duration` was `false` - do you think it needs its own explicit test/check?

Reply:
Yeah you're right — not sure what I was thinking here! That test seems fine to me